libpod/runtime_migrate_freebsd.go

```diff
@@ -0,0 +1,11 @@
+//go:build !remote
+
+package libpod
+
+func (r *Runtime) stopPauseProcess() error {
+	return nil
+}
+
+func (r *Runtime) Migrate(newRuntime string) error {
+	return nil
+}
```

Comment:
I don't think the full Migrate should be a NOP. It contains normal logic that should work on linux and bsd.

The only thing that causes an issue is the `r.stopPauseProcess()` call so only this should be a NOP on freebsd not the full Migrate()

Reply:
So we can use the same code for Migrate on both platforms? I misunderstood - partly because I'm not at all clear on what migrate does and if it is relevant to FreeBSD. I'll update the PR to try to share the Migrate logic.

Reply:
Migrate was originally added to convert all the container config to a new runtime AFAIK, i.e. `--new-runtime`. It basically stops all containers then rewrites the config. I.e. it was originally uses to convert from runc to crun.

If you only support one oci runtime on freebsd then this command has no real value right now, it will still stop all containers though.
However I see no reason why this code should be excluded from freebsd build, except the stopPauseProcess() call the rest should work.

Though we may add other db migrations in there at some point.

Reply:
I moved the Migrate method out of runtime_migrate_linux.go so that it can be shared by both platforms.